root/etc/confd/templates/config.xml.tmpl

```diff
@@ -0,0 +1,41 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I think those are (JENKINS_HOME, etc) are in the standard file anyway. Nothing that we use. If we used it it woudl be with `{{ getnev SOME_VAR }}` and we were not doing that.